sdk/core/Azure.Core/src/SerializableOptions.cs

```diff
@@ -4,19 +4,19 @@
 namespace Azure
 {
     /// <summary>
-    /// TODO
+    /// Provides the client options for serializing models.
     /// </summary>
     public class SerializableOptions
     {
         /// <summary>
         /// TODO
         /// </summary>
-        public bool IncludeReadOnlyProperties { get; set; } = true;
+        public bool IgnoreReadOnlyProperties { get; set; } = false;
```

Comment:
nit: the `= false` is not needed any  longer since that is the default for a bool